octoprint_mrbeam/analytics/analytics_handler.py

```diff
@@ -396,6 +401,11 @@ def add_engraving_parameters(self, eng_params):
 
     def add_cutting_parameters(self, cut_details):
         try:
+            cut_details.update(
+                {
+                    ak.Device.LaserHead.HEAD_MODEL_ID: self._laserhead_handler.get_current_used_lh_model_id(),
+                }
+            )
```

Comment:
please keep `map.update()` to larger or more dynamic changes than this
```py
cut_details[ak.Device.LaserHead.HEAD_MODEL_ID] = self._laserhead_handler.get_current_used_lh_model_id()
```

Reply:
I was thinking about this as well, but decided to do it with update if we need to add more in the future

Reply:
ok then I will change it